filebeat/module/logstash/slowlog/_meta/fields.yml

```diff
@@ -12,10 +13,12 @@
       description: >
         The module or class where the event originate.
     - name: thread
+      # Keyword?
       type: text
       description: >
         Information about the running thread where the log originate.
     - name: event
+      # event.original? or log.original?
```

Comment:
No, this is the "message" as logged by Logstash. So a very small part of the whole message as seen by Beats, in reality.

I'd leave here, and potentially have not indexed. Or stick to the convention of `keyword`, with a multi-field for `text`.

Reply:
Changed to `keyword` with `text` multi-field in https://github.com/elastic/beats/pull/10417.